src/test/scala/is/hail/io/LoadBgenSuite.scala

```diff
@@ -217,4 +217,13 @@ class LoadBgenSuite extends SparkSuite {
     hc.indexBgen(bgen)
     hc.importBgen(bgen, Option(sample)).count()
   }
+
+  @Test def testReIterate() {
+    val vds = hc.importBgen("src/test/resources/example.v11.bgen", Some("src/test/resources/example.sample"))
```

Comment:
You need to index the bgen file first. Or we can add the index to the repository.